src/GovUk.Education.ExploreEducationStatistics.Admin/Services/ThemeService.cs

```diff
@@ -159,6 +160,40 @@ public async Task<Either<ActionResult, Unit>> DeleteTheme(Guid themeId)
                 });
         }
 
+        public async Task<Either<ActionResult, Unit>> DeleteUITestThemes(CancellationToken cancellationToken = default)
+        {
+            return await _userService.CheckCanManageAllTaxonomy()
+                .OnSuccess(_ => _context.Themes
+                    .Where(theme => theme.Title.StartsWith("UI test theme") || theme.Title.StartsWith("Seed theme"))
```

Comment:
I'm not super thrilled about deleting things based on the _expectation_ that they'll match a specifically formatted `Title`, however this behaviour has remained unchanged, and I'd suggest that it would be better to remove outliers manually, rather than muddying the code with more string-based conditions.

Presumably these outliers were created either by an error or a non-standard approach, which has since been rectified, as the recent examples match the expected format. The ticket includes a criteria to tidy up dev, and I'm happy to remove these outliers manually.

Reply:
Actually I've just realised that the existing condition, `UI test theme`, matches almost no themes on `dev`, as the majority use the format "UI Test - Test theme - <timestamp> <some id>".

Reply:
So my views on this changed overnight :)

I think we push the cleaning up of random junk themes on dev not related to cleaning up UI test themes to another ticket.

`Actually I've just realised that the existing condition, UI test theme, matches almost no themes on dev`

I think matching by StartsWith `UI test theme` is actually fine. 

I think maybe for now we ignore the state of dev and just focus on what themes the UI tests will create once all the removing-topics work is done. If we decide now that the UI tests will create themes staring with `UI test theme`, then the issue is resolved. It just needs to be clear to the person who does the work to change the UI tests to create one theme per test run to name the theme created per run `UI test theme - {runId}`.

The theme naming pattern you mention (`UI Test - Test theme - <timestamp> <some id>`), AFAIK, isn't being created by the UI tests anymore, so I think we just ignore that for now.